package.json

```
@@ -38,24 +38,26 @@
     }
   },
   "peerDependencies": {
-    "eslint": ">= 6",
-    "prettier": ">= 1",
-    "eslint-plugin-import": "^2.20.0",
-    "eslint-plugin-prettier": "^3.1.0"
+    "eslint": ">= 8",
+    "prettier": ">= 2",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-prettier": "^4.2.1",
```

maybe make it more generic?
```suggestion
    "eslint-plugin-prettier": "^4.2.0",
```